plugins/woocommerce/src/Internal/DataStores/Orders/DataSynchronizer.php

```diff
@@ -0,0 +1,89 @@
+<?php
```

Comment:
this file could have more specific name I think, like `PostToCustomTableSynchronizer` or similar?

Reply:
The idea is that the class will handle anything related to data syncronization, this includes custom and post tables but also meta tables and perhaps other additional stuff that we haven't foreseen yet; so I think it's ok if the name isn't very specific. On the other hand being in the `Internal` namespace we can always rename it later if needed, even after being released.